demo/app.js

```diff
@@ -961,6 +991,18 @@ const Demo = React.createClass({
         />
         <br/><br/>
         <SearchInput />
+        <DisplayInput
+          hint='Click to Edit'
+          label='Display Input'
+          onBlur={this._handleInputStatusMessage}
+          onFocus={this._handleInputFocus}
+          onMouseOut={this._handleInputHideHint}
+          onMouseOver={this._handleInputShowHint}
```

Comment:
Ah, hadn't seen that you had `onMouseOut` and `onMouseOver` listeners added.  My previous comment about combing the two functions might not work depending on how you have all that setup.